00_Base/src/mapper/CitrineOcpiLocationMapper.ts

```diff
@@ -249,10 +259,10 @@ export class CitrineOcpiLocationMapper implements IOcpiLocationMapper {
     // TODO add remaining capabilities
     const capabilities: Capability[] = [];
 
-    if (authorizeRemoteStart === this.trueAttributeValue) {
+    if (authorizeRemoteStart === this.TRUE_ATTRIBUTE_VALUE) {
```

Comment:
Maybe in the future we can instead define an enum `VariableAttributeValues`.

Then this can read as `VariableAttributeValues.TRUE`.

Only thing is that I don't know if this would work since the values might have different types.

Reply:
upon another look this is overkill, i think i'll just keep the string nice and simple 😄 reverted back to a simple string and removed this const. 

agreed maybe in the future we could use some sort of enum if the values become more varied 🤔